### ISSUE TITLE
.asUrlString() attaches "/" to url 

### ISSUE THREAD

**Issue by nirsky** [evanshortiss/env-var#126]
Hey,

As the title says.
Not sure if this by design or a bug, but I think it's very confusing.

example:
```js
const MY_URL = env.get('BASE_URL').default('https://my.baseurl.com').asUrlString();
console.log(MY_URL); // https://my.baseurl.com/
```

**Comment by evanshortiss** [comment]
Internally `asUrlString()` uses the [url module from Node.js core](https://nodejs.org/api/url.html#url_constructor_new_url_input_base) (which implements WHATWG URL), and that adds the trailing slash. 

Here's an example from Node.js v14.3.0.

```js
new URL('http://a.b.com').toString()
'http://a.b.com/'
new URL('http://a.b.com/hello').toString()
'http://a.b.com/hello'
```

I feel like it makes sense to add the trailing slash, since in reality you're making a request to `/`. Perhaps this should be noted in the README?

**Comment by nirsky** [comment]
Probably worth mentioning. I spent a couple of minutes trying to understand why my request is failing.

**Comment by evanshortiss** [comment]
@nirsky gotcha, I think a PR in docs reminding developers to use the URL module/class when working with URLs is a good idea, e.g instead of:

```js
var path = '/thing'
var baseUrl = env.get('MY_URL').asUrlString()

// Could create invalid URL with two slashes!
var fullUrl = baseUrl + path
```

doing this:

```js
var path = '/thing'
var baseUrl = env.get('MY_URL').asUrlString()

// Will be correct, with no double forward slashes
var fullUrl = new URL(path, baseUrl).toString()
``` 

Would you like to PR it?

**Comment by evanshortiss** [comment]
@nirsky this will be addressed in docs in 6.2.0
